Tests/scripts/validate_premium_packs.py

```diff
@@ -0,0 +1,266 @@
+        ids_match = log_message_if_statement(statement=(server_pack["id"] == index_pack["id"]),
+                                             error_message=f'server pack id {server_pack["id"]} '
+                                                           f'does not match index pack id {index_pack["id"]}')
+        if ids_match:
+            prices_match = log_message_if_statement(statement=(server_pack["price"] == index_pack["price"]),
+                                                    error_message=f'server pack price {server_pack["price"]} '
+                                                                  f'for pack id {server_pack["id"]} '
+                                                                  f'does not match the pack price '
+                                                                  f'found in the index file {index_pack["price"]}',
+                                                    success_message=f'Pack: {server_pack["id"]} is valid.')
+            if not prices_match:
+                packs_identical = False
+        else:
+            packs_identical = False
+
+    return packs_identical
+
+
+def check_commit_in_master_history(index_commit_hash, master_history_path):
+    """Assert commit hash is in master history.
+
+    Args:
+        index_commit_hash: commit hash
+        master_history_path: path to a file with all the master's commit hash history separated by \n
+
+    Returns: True if commit hash is in master history, False otherwise.
+    """
+
+    with open(master_history_path, 'r') as master_history_file:
+        master_history = master_history_file.read()
+        master_commits = master_history.split('\n')
+
+    return log_message_if_statement(statement=(index_commit_hash in master_commits),
+                                    error_message=f'Commit hash {index_commit_hash} is not in master history',
+                                    success_message="Commit hash in index file is valid.")
+
+
+def main():
+    install_logging('Validate Premium Packs.log')
+    logging.info('Retrieving the index file')
+    options = options_handler()
+    index_file_path = unzip_index_and_return_index_file(options.index_path)
+
+    # Validate index.json file
+    index_is_valid, index_data = check_and_return_index_data(index_file_path)
+
+    # Validate commit hash in master history
+    commit_hash_is_valid = check_commit_in_master_history(index_data["commit"], options.master_history)
+
+    if (not index_is_valid) or (not commit_hash_is_valid):
+        logging.debug('Index content is invalid. Aborting.')
+        os.remove(index_file_path)
+        sys.exit(1)
+
+    # Get the host by the ami env
+    hosts, _ = Build.get_servers(ami_env=options.ami_env)
+
+    logging.info('Retrieving the credentials for Cortex XSOAR server')
+    secret_conf_file = get_json_file(path=options.secret)
+    username: str = secret_conf_file.get('username')
+    password: str = secret_conf_file.get('userPassword')
+
```

Comment:
Create a different method that will return the username and password only, or even better extract the username and password in the shell script and pass them as parameters.
code example:
```
SECRET_CONF_PATH=$(cat secret_conf_path)

USERNAME=$(cat $SECRET_CONF_PATH | jq '.username')
PASSWORD=$(cat $SECRET_CONF_PATH | jq '.userPassword')

# remove quotes from password
temp="${PASSWORD%\"}"
temp="${temp#\"}"
PASSWORD=$temp

# remove quotes from username
temp="${USERNAME%\"}"
temp="${temp#\"}"
USERNAME=$temp
```
no need to save all the `secret_conf_file` in main() frame just for those

Reply:
I disagree. 
1. Python is easier for debugging
2. I think it's safer not to save the username and password inside the docker shell parameters. It's a little easier to get that way.